README.md

```diff
@@ -36,72 +36,91 @@ You should be able to delete any article **you created** from the database.
 
 >Note: if you try to delete an article you did not write, it won't work but you won't recieve any error message.
 
-## Setup
-You need to set up the frontend and backend applications to test the server. Here is a quick guide after cloning the repository:
+## Deployment
+You need to set up the frontend and backend applications to test the server. 
+For deployment, `development` and `production` modes are available
 
-### Backend
+> ##### Notes for the iScsc members:
+> Send me a message and I'll send you back a crypted version of the official `.env.production` and `.env.development` files.
+
+Here is a quick guide after cloning the repository:
+
+### Development mode
+#### .env file
+Before deploying the application, you need to set the environement variables
 From the root directory of the repository, do the following:
-```bash
-cd backend
-npm install
-cp .env.example .env
 ```
+cd config
+cp .env.example .env.development
+```
+After copying the example config of `.env`, you must fill in the missing information in this file. Check the example for more information.
 
-After copying the example config of `.env`, you must fill in the missing information in this file:
-- The port to deploy the backend
-- The database name
-- The credential for the database
-- The secret key used to sign JWT
-- The expiration time for JWT
-- The maximum size of articles
-
-If you don't know how to deploy your database, consider using [Atlas](https://www.mongodb.com/atlas/database).
+> If you don't know how to deploy your database, consider using [Atlas](https://www.mongodb.com/atlas/database).
 
-> #### Notes for the iScsc members:
-> To access the official iScsc database, send me a message and I'll send you back a crypted version of the official `.env`.
+#### Frontend
+From the root directory of the repository, do the following:
+```
+cd frontend
+npm install
+npm run dev
+```
 
- To deploy the server once your `.env` is ready, run 
+#### Backend
+From the root directory of the repository, do the following:
 ```bash
-node .
+cd backend
+npm install
+npm run start
 ```
 
-If everything is good, the application should output:
-```
-Server listening: http://localhost:<PORT>
-```
+> You will need `nodemon` to run the backend
 
-### Frontend
-From the root directory of the repository, do the following:
+### Production mode
+The production mode allows to deploy the application on the server. To use it, you will need:
+- `docker`
+- `docker-compose`
+
+#### .env file
+Before deploying the application, you need to set the environement variables, as for `developement` mode.
 ```
-cd frontend
-npm install
-npm start
+cd config
+cp .env.example .env.production
 ```
-It will start the application on a localhost port that you will be able to access in your browser.
-Make sure the backend application is running before launching the frontend.
+
+#### Docker
+Once you `.env.production` is ready, run
+`sudo docker-compose up`
```

Comment:
as discussed in private and with the current location of the config files, this should be
```bash
sudo docker-compose --env-file config/.env.production up
```

Reply:
or move the config files from `./config/` to `./` :yum: :+1:

Reply:
I moved the config files in [33229c7](https://github.com/iScsc/iscsc.fr/pull/20/commits/33229c7bb87594aa207832e5353558b9270b1761)

Reply:
And for the first point, [db9c5ef](https://github.com/iScsc/iscsc.fr/pull/20/commits/db9c5ef3d534fef6096315c049a10ac806c6c741) specify the new path